crypto/keys/der/der_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package der
+
+import (
+	"bytes"
+	"encoding/pem"
+	"testing"
+
+	"github.com/google/trillian/testonly"
+)
+
+func MustMarshalPublicPEMToDER(keyPEM string) []byte {
```

Comment:
In fact, rather than declaring this func, you could use the func declared at https://github.com/google/trillian/blob/master/crypto/keys/testonly/keys.go#L37. You'll just need to change this file's package to "der_test", then dot-import the "der" package, to avoid an import cycle.